readme/api/references/plugin_manifest.md

```diff
@@ -37,8 +37,10 @@ Name | Type | Required? | Description
 
 | Properties | Description |
 | --- | --- |
-| src | a relative path to src dir. |
-| label | description of the image. |
+| src | a path or URL to an icon. If a path, `src` should either be relative to the root of the repository (e.g. `screenshots/a.png`) or relative to the `src` directory (e.g. `../screenshots/a.png`). |
```

Comment:
"to a screenshot"

Reply:
Actually since there are very few plugins that have screenshots at the moment, it would make sense to enforce only one option. Maybe relative to the root of the repository then? That would make their location more predictable by future tools

Reply:
> Maybe relative to the root of the repository then? That would make their location more predictable by future tools

I've updated `plugin_manifest.md` to reflect this. One potential concern is that plugins created with an older version of the generator will still try to enforce screenshots being relative to `src/`. This might be confusing to authors attempting to add screenshots to their plugins.

As such, it might also make sense to include a note that suggests updating the plugin template.